packages/url/src/index.js

```diff
@@ -3,9 +3,21 @@
  */
 import { parse, stringify } from 'qs';
 
+const URL_REGEXP = /^(?:https?:)?\/\/\S+$/i;
 const EMAIL_REGEXP = /^(mailto:)?[a-z0-9._%+-]+@[a-z0-9][a-z0-9.-]*\.[a-z]{2,63}$/i;
 const USABLE_HREF_REGEXP = /^(?:[a-z]+:|#|\?|\.|\/)/i;
 
+/**
+ * Determines whether the given string looks like a URL.
+ *
+ * @param {string} url The string to scrutinise.
+ *
+ * @return {boolean} Whether or not it looks like a URL.
+ */
+export function isURL( url ) {
```

Comment:
Nice util, I think we may be able to use it in:
https://github.com/WordPress/gutenberg/blob/f9da76a18f1dda3b0aaa00218b49914901c6ab0e/packages/editor/src/components/rich-text/index.js#L285
Removing a duplicate regex.

Reply:
I've already replaced that regex 🙂